boards/arm/nrf9160_innblue11/doc/index.rst

```diff
@@ -0,0 +1,203 @@
+.. _nrf9160_innblue11:
+
+nRF9160-INNBLUE11
+################
+
+Overview
+********
+
+The nRF9160 innblue v11 is a single-board development kit for evaluation
+and development on the nRF9160 SiP for LTE-M and NB-IoT. The
+nrf9160_innblue11 board provides support for the Nordic Semiconductor
+nRF9160 ARM Cortex-M33F CPU with ARMv8-M Security Extension and the
+following devices:
+
+* :abbr:`ADC (Analog to Digital Converter)`
+* CLOCK
+* FLASH
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`I2C (Inter-Integrated Circuit)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* :abbr:`RTC (nRF RTC System Clock)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter with EasyDMA)`
+* :abbr:`WDT (Watchdog Timer)`
+* :abbr:`IDAU (Implementation Defined Attribution Unit)`
+
+.. figure:: img/nrf9160_pca10090.png

```

Comment:
Change to reference the right image name:  ``img/nrf9160_innblue11.png`` and update the :alt: text and the figure title (``nRF9160 PCA10090 DK (Credit: Nordic Semi)``) to refer to this nrf9160 innblue11 board.